src/auth/structs.rs

```diff
@@ -68,7 +68,7 @@ impl Context {
 impl Default for Context {
     fn default() -> Self {
         Self {
-            variant: ContextVariant::Activated,
```

Comment:
I think default should be activated and notactivated the "special" case.
Just to be sure that as a bare minimum the user is activated